index.js

```diff
@@ -21,7 +28,9 @@ function intoRels (acc, x) {
 function createObjects (acc, p) {
   // rel="next" => 1: rel 2: next
   var m = p.match(/\s*(.+)\s*=\s*"?([^"]+)"?/)
-  if (m) acc[m[1]] = m[2];
+  if (m) {
+    acc[m[1]] = m[2];
+  }
```

Comment:
line 14 is fine for two reasons:
1. it's a nested if, so you could argue you'd want curlys like the other if
2. and more importantly it didn't just change already existing code for no good reason ;)